services/workflows-service/prisma/migrations/20240110184558_add_transition_schema_to_workflow_definition/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "WorkflowDefinition" ADD COLUMN     "transitionSchema" JSONB;
```

Comment:
What's `transitionSchema`?

Reply:
Transition schema will be used to validate context data between transitions or initial.(Only for creation currently).